### PR TITLE
More documentation

### PR DESCRIPTION
**Pull request by timperrett**

### CODE REVIEW

File: docs/src/hugo/content/introduction/competitors.md [hunk 1]
```diff
@@ -2,30 +2,24 @@
...
+  main:
+    parent: 'intro'
+    identifier: intro-competitors
+    url: /introduction/competitors.html
+    weight: 3
 ---
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas eu commodo nisi. Etiam sagittis enim purus, id tristique ante tincidunt id. Nunc placerat velit neque. Integer finibus velit nec vestibulum elementum. Mauris vel tempor velit. Nam sodales malesuada purus vel bibendum. Sed eu mauris sit amet nulla sodales imperdiet. Sed pulvinar iaculis magna at feugiat. Quisque luctus nec lectus vitae pharetra. Cras euismod pharetra augue ac consequat. Integer id euismod lectus, non cursus ligula. Nunc lorem dolor, condimentum nec nunc non, hendrerit ullamcorper tellus. Sed tortor nibh, mollis eget euismod ac, tempor a eros. Etiam feugiat odio ac interdum pellentesque. Nam dignissim, ipsum ut vehicula interdum, justo dui accumsan felis, in maximus diam justo tristique tellus.
-
 ## Spinnaker
```
Comment (timperrett) [inline docs/src/hugo/content/introduction/competitors.md]: @okoye any chance you can read this modified section and provide a comment?

File: docs/src/hugo/content/introduction/competitors.md [hunk 2]
```diff
@@ -2,30 +2,24 @@
...
+Spinnaker has support for a wide range of process workflows (for example, gating deployments on manual intervention). Broadly speaking, Nelson is strongly against this from a philosohpical standpoint, as such manual processes are a crutch for a lack of automation of continuous testing. Nelson by comparison provides a foundation for a robust experimentation system, where testing in production is the norm - this embraces the reality of the world we live in: ship frequently, experiment often, observe everything. The underlying intent here is that Nelson in no way perscribes how you will operate your release management; it simply provides a set of primitives.
```
Comment (adelbertc) [inline docs/src/hugo/content/introduction/competitors.md]: Typo - "philosophical"

File: docs/src/hugo/content/introduction/competitors.md [hunk 3]
```diff
@@ -2,30 +2,24 @@
...
+The [Collins](https://tumblr.github.io/collins/) configuration management system - along with other configuration management solutions - do not truley embrace immutable infrastructure. Typically these systems are focused on maintaining a static fleet of servers and evolving them over time, applying patches and so forth. This is an intrinsically different model to the one Nelson provides. Collins would be a better fit for teams that which to manage the underlying infrastructure a scheudler might run on, instead of end-user applications.
```
Comment (adelbertc) [inline docs/src/hugo/content/introduction/competitors.md]: Typo - "scheduler"

File: docs/src/hugo/content/introduction/competitors.md (hunk 1 above)
Comment (okoye) [reply docs/src/hugo/content/introduction/competitors.md]: @timperrett sure thing, I'll do that on my flight to Pittsburgh

File: docs/src/hugo/content/introduction/competitors.md [hunk 4]
```diff
@@ -2,30 +2,24 @@
 layout: "single"
 title: "Competitors"
 preamble: >
-  Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas eu commodo nisi. Etiam sagittis enim purus, id tristique ante tincidunt id. Nunc placerat velit neque. Integer finibus velit nec vestibulum elementum.
+  Nelson is in many ways a uniuqe system; the principles and methods used in its construction are not common in the operations domain. However, is is common for potential users to draw parrallels with other systems, and navigating the competitive space can often be difficult. The systems most commonly compared with Nelson are enumerated below a brief comparitive explanation is provided to help understand Nelson's value proposistion.
 contents:
 - Spinnaker
 - Collins
```
Comment (okoye) [inline docs/src/hugo/content/introduction/competitors.md]: You should add plain ole Jenkins too ;)

File: docs/src/hugo/content/introduction/competitors.md [hunk 5]
```diff
@@ -2,30 +2,24 @@
...
-
-## Collins
+One of the most common systems Nelson is compared to is the [Spinnaker project](https://spinnaker.io). On the face of it, the systems are indeed solving similar problems. The truth is however a little more nuanced. Spinnaker was born out of Netflix as a replacement for their [Asguard deployment system](https://github.com/Netflix/asgard). The primary difference between Spinnaker and Nelson is that Nelson is container-first where as Spinnaker was primarily designed to operate with Netflix's virtual machine "bakery". Whilst they have since retrofitted support for containers, the approach of the project cannot detech itself from that legacy workflow easily. Given Nelson is focused on integration with scheduling systems as its common substrate for deployment, the management of the virtual machines underlying those schedulers is out of scope. Indeed, with the prevelance of hosted scheudling systems like [EKS](https://aws.amazon.com/eks/), [EKS](https://cloud.google.com/kubernetes-engine/) and [AKE](https://azure.microsoft.com/en-us/services/kubernetes-service/), the need for even running your own machines appears to be passing.
```
Comment (okoye) [inline docs/src/hugo/content/introduction/competitors.md]: Took a look, I'll rephrase some parts of this (in flight tomorrow) and send it over when I land.

**Pull request merged**